bouncer/tests/btc_vault_swap.ts

```diff
@@ -112,8 +114,25 @@ async function testVaultSwap(depositAmountBtc: number, brokerUri: string, destin
   testBtcVaultSwap.log(`Balance increased, Vault Swap Complete`);
 }
 
+async function openPrivateBtcChannel(brokerUri: string) {
+  // TODO: Use chainflip SDK instead so we can support any broker uri
+  assert.strictEqual(brokerUri, '//BROKER_1', 'Support for other brokers is not implemented');
+
+  // TODO: use chainflip SDK to check if the channel is already open
+  try {
+    await jsonRpc('broker_open_private_btc_channel', [], 'http://127.0.0.1:10997');
+    testBtcVaultSwap.log('Private Btc channel opened');
+  } catch (error) {
+    // We expect this to fail if the channel already exists from a previous run
+    testBtcVaultSwap.debugLog('Failed to open private Btc channel', error);
```

Comment:
If this is the expectation when we fail then I think we should check that the channel already exists, or inspect the error so we know etc.
else if we fail because the request params are wrong, this function will still pass

Reply:
yes, thats why i put a TODO there to check if its open once we use the chainflip SDK.